<?xml version="1.0"?>
<launch>
  <node pkg="ball_picker" type="positioner" name="positioner" output="screen">
    <remap from="depth_rect" to="/cam3d/depth/image_raw" />
    <remap from="cam_info" to="/cam3d/depth/camera_info" />
    <remap from="goal_coords" to="/goal_coords" />
    <remap from="goal_angle" to="/goal_angle" />
    <remap from="costmap_custom_obstacles" to="/costmap_custom_obstacles" />
    <remap from="kinect_servo" to="/kinect_servo_repeater" />
    <!--
         <remap from="kinect_servo" to="/dynamixel/kinect_controller/command" />
    -->
  </node>
</launch>
